react/features/connection-indicator/components/web/ConnectionIndicatorIcon.js

```diff
@@ -47,36 +43,8 @@ export const ConnectionIndicatorIcon = ({
     colorClass,
     connectionIndicatorInactiveDisabled,
     isConnectionStatusInactive,
-    isConnectionStatusInterrupted,
-    track
+    isConnectionStatusInterrupted
 }: Props) => {
-    const sourceNameSignalingEnabled = useSelector(state => getSourceNameSignalingFeatureFlag(state));
```

Comment:
I'd rather keep large video and thumbnail impls as a legacy solution with a TODO to be converted for using the hook instead

Reply:
Discussed with William and will change to keep track streaming status locally.

Reply:
It turns out to be more complicated than expected to change code to use local state for track streaming status. It's better to focus this PR on fixing the bug with minimal change. Thus will keep using redux store for track streaming status. Converting to local state can be another PR.

Reply:
I removed this code from this file because `ConnectionIndicatorIcon.js` is always part of `Thumnail.js` thus not necessary to have it's own code to add/remove listener.
However if we really want to add such code to every where track streaming status is used, we should also add that to `ConnectionIndicatorContent.js`.